### PR TITLE
Prepare 2.8.3 release

### PR DESCRIPTION
**Pull request by vmatyusGitHub**
# Proposed 2.8.3 release

# Changes

# 2.8.3 (2022-04-04)
- [UPGRADED] `@ibm-cloud/cloudant` dependency to version `0.1.1`.
- [NOTE] Move `retry-axios` and `ibm-cloud-sdk-core` to peerDependencies.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# Unreleased
+# 2.8.3 (2022-04-04)
```
Comment (ricellis) [inline CHANGES.md:1]: Let's bump this to tomorrow to give time to run the large regression test suite before doing the release
```suggestion
# 2.8.3 (2022-04-05)
```

**Review by ricellis — APPROVED** [review]
+1, but given the dependabot changes and time for the large test suite we should move out a day

File: CHANGES.md (hunk 1 above)
Comment (vmatyusGitHub) [reply CHANGES.md]: Rebased in 8620476

**Comment by vmatyusGitHub** [comment]
Thank you for reviewing! I am about to merge this.

**Pull request merged**